setup.py

```diff
@@ -11,7 +11,7 @@
       py_modules=['tap-appstore'],
       install_requires=[
           'singer-python==5.2.3',
-          'appstoreconnect==0.3.0',
+          'appstoreconnect==0.9.0',
```

Comment:
picks up a bunch of bugfixes https://github.com/Ponytech/appstoreconnectapi/blob/master/CHANGELOG.md